src/Themes/Fork/templates/Frontend/base/Head.html.twig

```diff
@@ -31,7 +31,7 @@
   {# Google Fonts #}
   <link href="https://fonts.googleapis.com/css?family=Courgette|Roboto:300,400,500,700,900" rel="stylesheet">
 
-  {% for css_file in css_files %}
+  {% for css_file in cssFiles %}
```

Comment:
Dit snap ik niet? variabelen moeten toch met _ zijn? Lijkt me dat je het in PHP moet aanpassen.

Reply:
@bjorvack I am following @tijsverkoyen in this as well; it had been bugging me throughout this review. I'm not too fond of the inconsistency, either all with _ or all camel case